forAndroid/captivewebview/build.gradle

```diff
@@ -23,8 +22,8 @@ android {
     }
 
     defaultConfig {
-        minSdkVersion 21
-        targetSdkVersion 28
+        minSdkVersion 26
```

Comment:
Is this jump in the minSdkVersion needed?
The Android SKD min SDK version is still 19.

Reply:
The WebChromeClient class, at least, was updated to utilise some of the most recent interfaces, here:
https://github.com/vmware/captive-web-view/pull/8/commits/c61a34f54c9e361520c4b093e282c8e5e24b7a74